### PR TITLE
sops: update page

### PR DESCRIPTION
**Pull request by Shensen1**
Hi, 
my first public contribution here, so greetings to my Ma! :)

I think this one is essential, got stuck twice now by remembering this one and so I guess it should be shared with others 

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/sops.md
Comment (Magrid0) [inline pages/common/sops.md:14]:
```suggestion
- Update the declared keys in .sops.yaml:
```

Comment (Magrid0) [reply pages/common/sops.md:14]: Hi, welcome to TlDr, as the bot said you have to add a : at the end of each command description

**Review by Magrid0 — CHANGES_REQUESTED** [review]

File: pages/common/sops.md
Comment (Magrid0) [inline pages/common/sops.md:16]:
```suggestion
`sops updatekeys {{/path/to/file.yaml}}`
```

And also add {{ }} around the file path

**Review by Magrid0 — CHANGES_REQUESTED** [review]

**Comment by Magrid0** [comment]
> Ok, nice So now just wait for another ok and then it gets pushed?

Yep!

> But why have the file paths to be in that format? actually wanted to add one simpler syntax example

Because the [Style Guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#example-commands) say they have to be written in this way

image

**Comment by Shensen1** [comment]
Ok, makes sense, head [guidelines](https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines) in mind with

2. When in doubt, keep new command-line users in mind. Err on the side of clarity rather than terseness. For example, commands that require sudo should include it directly in the examples.

**Review by kbdharun — APPROVED** [review]
Welcome to tldr and thanks for your contribution. LGTM, after @sebastiaanspeck's suggestion above.

**Review by Magrid0 — APPROVED** [review]
LGTM, i think it'll be ready to merge after sebastiaanspeck suggestion

**Comment by spageektti** [comment]
Since the author of this PR has been inactive for a long time, I have made the suggested changes.

[Maintainers Guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/maintainers-guide.md#ii-handling-prs) says:

> If the only issues holding up a merge are **trivial fixes**
>   (typos, syntax errors, etc.), and the author doesn't respond in a day or two,
>   **maintainers can make the necessary changes themselves**,
>   and proceed with the merge process.

**Review by spageektti — APPROVED** [review]

File: pages/common/sops.md
Comment (kbdharun) [inline pages/common/sops.md:1]: Just now noticed the page description looks a bit vague, maybe we could reword it to something more descriptive.

**Review by kbdharun — APPROVED** [review]

**Pull request merged**
